pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -192,25 +198,19 @@ func (c *clusterMemberRemovalController) attemptToScaleDown(ctx context.Context,
 				unhealthyMembersURLs = append(unhealthyMembersURLs, unhealthyMember.Name)
 			}
 		}
-		if len(unhealthyVotingMemberMachinesPendingDeletion) > 0 {
-			klog.V(4).Infof("found unhealthy voting members with machine pending deletion: %v", unhealthyVotingMemberMachinesPendingDeletion)
-			klog.V(4).Infof("unhealthy members found: %v", unhealthyMembersURLs)
-		} else {
-			return fmt.Errorf("cannot proceed with scaling down, unhealthy voting members found: %v, none are pending deletion", unhealthyMembersURLs)
+		if len(unhealthyVotingMemberMachinesPendingDeletion) <= 0 {
+			klog.V(2).Infof("cannot proceed with scaling down, unhealthy etcd members found: %v", unhealthyMembersURLs)
+			return fmt.Errorf("cannot proceed with scaling down, unhealthy etcd members found: %v", unhealthyMembersURLs)
```

Comment:
Slightly confusing since `unhealthyVotingMemberMachinesPendingDeletion <= 0` wouldn't make sense as it's the length of a slice which can't be < 0. So it should be `unhealthyVotingMemberMachinesPendingDeletion == 0`.

Also can we update the log and error statement to `cannot proceed with scaling down, unhealthy voting etcd members found: %v but none are pending deletion`.

Since its not that we can never scale-down when unhealthy members are present, but rather they need to be pending deletion first. Plus making it clear that they are voting members since this doesn't apply to learners.